README.md

```diff
@@ -5,61 +5,103 @@
 [![Total Downloads](https://img.shields.io/packagist/dt/muffin/obfuscate.svg?style=flat-square)](https://packagist.org/packages/muffin/obfuscate)
 [![License](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](LICENSE)
 
-CakePHP 3 support for ID obfuscation using any combination of HashIds, Optimus and/or Tiny.
+Primary key obfuscation for CakePHP 3 using any combination of
```

Comment:
I am not too sure about using the word "combination". Most people would only use either one of them.